techniques/smil/SM11.html

```diff
@@ -2,76 +2,65 @@
+         <p>この達成方法 (参考) の使用法と、この達成方法が WCAG 2.1 達成基準 (規定) とどのように関係するのかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) のセクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法の存在は、その技術があらゆる状況で WCAG 2.1 を満たすコンテンツを作成するために使用できることを意味するものではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p> Applies to SMIL 1.0 </p>
-            <p>This technique relates to:</p>
+            <h2>適用 (対象)</h2>
+            <p> SMIL 1.0 </p>
+            <p>これは、次の達成基準に関連する達成方法である:</p>
             <ul>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-prerecorded">Success Criterion 1.2.2: Captions (Prerecorded)</a> (Sufficient as a way to meet <a href="../general/G87">G87: Providing closed captions</a>)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-prerecorded">達成基準 1.2.2: キャプション (収録済) </a> (<a href="../general/G87">G87: クローズドキャプションを提供する</a>の達成方法として十分)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-live">Success Criterion 1.2.4: Captions (Live)</a> (Sufficient as a way to meet <a href="../general/G9">G9: Creating captions for live synchronized media</a>)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-live">達成基準 1.2.4: キャプション (ライブ) </a> (<a href="../general/G9">G9: ライブの同期したメディアに対してキャプションを作成する</a>の達成方法として十分)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-live">Success Criterion 1.2.4: Captions (Live)</a> (Sufficient as a way to meet <a href="../general/G87">G87: Providing closed captions</a>)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-live">達成基準 1.2.4: キャプション (ライブ) </a> (<a href="../general/G87">G87: クローズドキャプションを提供す</a>の達成方法として十分の達成方法として十分)
                </li>
-               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-live">Success Criterion 1.2.4: Captions (Live)</a> (Sufficient as a way to meet an unwritten technique)
+               <li><a href="https://waic.jp/docs/WCAG21/Understanding/captions-live">達成基準 1.2.4: キャプション (ライブ) </a> (書かれていない達成方法を満たす手段として十分)
                </li>
             </ul>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p> The objective of this technique is to provide a way for people who are deaf
-               or otherwise have trouble hearing the dialogue in audio visual material to be
-               able to view the material. With this technique all of the dialogue and
-               important sounds are available in a text stream that is displayed in a
-               caption area. 
+            <h2>解説</h2>
+            <p> この達成方法の目的は、耳が聞こえないか、またはそうでなくとも、視聴覚コンテンツにおいて発話を聞くことに問題のある利用者に、コンテンツを閲覧できる方法を提供することである。この達成方法では、発話と重要な音の全てがキャプション領域に表示される手話通訳の映像によって入手できる。
```

Comment:
deaf ＝ろう としてください

or otherwise
又は別の方法で
https://eow.alc.co.jp/search?q=or+otherwise

have trouble ～ing
～するのが困難な
https://eow.alc.co.jp/search?q=have+trouble

dialogueは会話でよい気はします。
https://eow.alc.co.jp/search?q=dialogue

text streamはタイトルでテキストストリームとしているので合わせましょう

Reply:
申し訳ありいません。SM14の和訳が貼り付けてあったようです。SM11に差し替えた上で指摘を反映しました。

Reply:
> deaf ＝ろう としてください

この指摘、SM14にも当てはまると思います。別途修正プルリクを出そうと思います。

Reply:
@8845musign 

> この指摘、SM14にも当てはまると思います。

出してもらってもよいのですが、SMIL 1.0や2.0というおそらく誰も見向きもしない仕様にトドメを刺すべきだと思うんですよね…。